calculator/operations.py

```diff
@@ -1,23 +1,23 @@
+from functools import reduce
+import operator as op
+from sympy import symbols
+from sympy.plotting import plot
+from sympy.parsing.sympy_parser import parse_expr
 
 def add(*args):
-    # your implementation here
-    pass
+    return sum(args)
 
 def subtract(*args):
-    # your implementation here
-    pass
+    return reduce(op.sub, args)
 
 def multiply(*args):
-    # your implementation here
-    pass
+    return reduce(op.mul, args)
 
 def divide(*args):
-    # your implementation here
-    pass
+    return reduce(op.truediv, args)
 
-def plot(*args):
-    # OPTIONAL EXTRA CREDIT FUNCTION! 
-    # See README for info.
-    pass
-
-# add your custom operations here
+def plot_op(expr, a, b):
```

Comment:
Great job making this work. I guess maybe try thinking about how a unittest for this can be written.